package.json

```diff
@@ -87,6 +87,7 @@
     "nsp": "nsp check",
     "posttest": "npm run lint",
     "test": "make bootstrap_zip && mocha test/",
+    "test:server": "mocha --recursive test/server/unit/",
```

Comment:
I'm happy to just back this line out and figure out the details later. I"ll file a followup bug.